### PR TITLE
[DUOS-1483][risk=no] Fix user profile bugs

### PR DESCRIPTION
**Pull request by rushtong**
## Addresses
Fixes https://broadworkbench.atlassian.net/browse/DUOS-1483

## Changes
* Save user on "Continue later"
* Use user email as the required field in academic business email
* Remove the required from academic/business since it will be defaulted to email
* Clarified requirements for NIH eRA Commons ID
* Removed conditional formatting for Continue Later/Submit buttons

----
Have you read [Terra's Contributing Guide](https://github.com/DataBiosphere/terra-ui/wiki/Contributor-Guide) lately? If not, do that first.

- Label PR with a Jira ticket number and include a link to the ticket
- Label PR with a security risk modifier [no, low, medium, high]
- PR describes scope of changes
- Get a minimum of one thumbs worth of review, preferably two if enough team members are available
- Get PO sign-off for all non-trivial UI or workflow changes
- Verify all tests go green
- Test this change deployed correctly and works on dev environment after deployment

### CODE REVIEW

File: src/pages/ResearcherProfile.js [hunk 1]
```diff
@@ -177,7 +177,7 @@ export const ResearcherProfile = hh(class ResearcherProfile extends Component {
       isThePI = false,
       piEmail = false,
       piName = false,
-      showValidationMessages = false;
+      showValidationMessages = true;
```
Comment (JVThomas) [inline src/pages/ResearcherProfile.js]: You should switch this back to false. The value will flip to true through the validation conditionals below (if the target value is found to be invalid)

Comment (MiriamAlter) [reply src/pages/ResearcherProfile.js]: I thought I undid that change. Good catch.

File: src/components/eRACommons.js [hunk 2]
```diff
@@ -126,7 +126,7 @@ export const eRACommons = hh(class eRACommons extends React.Component {
 
     return (
       div({ className: this.props.className }, [
-        label({ className: 'control-label' }, ['NIH eRA Commons ID*']),
+        label({ className: 'control-label' }, ['NIH eRA Commons ID ', span({ className: 'italic display-inline' }, ['(Required for DARs)'])]),
```
Comment (rushtong) [inline src/components/eRACommons.js]: We should revert this change for now. The message displayed is true, but this component is also displayed on the DAR application page so adding this text here makes it slightly more confusing there since you're already making a DAR. 

What we should do, in a separate story/PR, is to update this component to take a title as a prop. The places that use this can then provide whatever title text that makes sense for their usage.

Lastly, the term `DAR` is something we use internally to refer to a `Data Access Request`, so using that term here would be confusing to some users, especially international ones.

File: src/pages/ResearcherProfile.js [hunk 3]
```diff
@@ -622,7 +622,7 @@ export const ResearcherProfile = hh(class ResearcherProfile extends Component {
               div({ className: 'form-group' }, [
                 div({ className: 'col-xs-12', style: { 'marginTop': '20px' } }, [
                   label({ className: 'control-label rp-title-question default-color' }, [
-                    'Researcher Identification*',
+                    'Researcher Identification',
```
Comment (rushtong) [inline src/pages/ResearcherProfile.js]: I think we should revert this change as well. The explanation text does a good job of explaining what is required and what isn't.

**Review by rushtong — COMMENTED** [review]
Nice work, please see comments inline.

**Review by rushtong — COMMENTED** [review]
Changes look good! 👍🏽 Side note, I can't approve this since I created it originally. Feel free to merge when you're ready.

**Review by MiriamAlter — APPROVED** [review]
Leaving a review to enable merging
